Frameworks/CoreText/CTFont.mm

```diff
@@ -101,25 +110,74 @@
 const CFStringRef kCTFontFeatureSelectorSettingKey = static_cast<CFStringRef>(@"kCTFontFeatureSelectorSettingKey");
 
 /**
+ All these values are fixed number in IOS no matter on iphone* or ipad*.
+ */
+const CGFloat kCTFontLabelFontSize = 17.0f;
+const CGFloat kCTFontButtonFontSize = 14.0f;
+const CGFloat kCTFontSmallSystemFontSize = 12.0f;
+const CGFloat kCTFontSystemFontSize = 14.0f;
+
+using namespace Microsoft::WRL;
+
+struct __CTFont {
```

Comment:
+1

---
In reply to: [78205733](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78205733) [](ancestors = 78205733)